model/document.js

```diff
@@ -197,7 +197,32 @@ module.exports = memoize(function (db) {
 			if (statusLog.length) data.statusLog = statusLog;
 
 			return data;
-		} }
+		} },
+		toWSSchema: {
+			value: function (ignore) {
+				var schema = {
+					type: "array",
+					items: {
+						type: "object",
+						properties: {
+							code: {
+								type: "enum",
+								ref: "documents"
+							},
+							data: {}
+						}
+					}
+				};
+				schema.items.properties.files =
+					this.files.map._descriptorPrototype_.type.prototype.toWSSchema();
+				schema.items.properties.owner = { type: "string" };
+				if (this.dataForm.constructor !== this.database.FormSectionBase) {
+					schema.items.properties.data = this.dataForm.toWSSchema();
+					schema.items.dataForms = [ { title: "", properties: [] } ];
```

Comment:
Looks a bit odd. Is this an attempt to get information about section in the schema? If, so then we should either fill it in with sth interesting, or remove this altogether.